container-search/src/main/java/com/yahoo/prelude/query/Item.java

```diff
@@ -60,7 +60,8 @@
         PREDICATE_QUERY(23),
         REGEXP(24),
         WORD_ALTERNATIVES(25),
-        NEAREST_NEIGHBOR(26);
+        NEAREST_NEIGHBOR(26),
+        LOCATION_TERM(27);
```

Comment:
Need to update abi
`[ERROR] Class com.yahoo.prelude.query.Item$ItemType: Extra field public static final enum com.yahoo.prelude.query.Item$ItemType LOCATION_TERM
`